pkg/cmd/marathon/marathon_about_test.go

```diff
@@ -0,0 +1,50 @@
+package marathon
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/dcos/dcos-cli/pkg/config"
+	"github.com/dcos/dcos-cli/pkg/mock"
+	"github.com/dcos/dcos-core-cli/pkg/marathon"
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMarathonAbout(t *testing.T) {
+	type info struct {
+		FrameworkID string `json:"frameworkId"`
+	}
+	marathonInfo := info{
+		FrameworkID: "marathonId",
+	}
+
+	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		assert.Equal(t, "/service/marathon/v2/info", r.URL.String())
+		assert.Equal(t, "GET", r.Method)
+		assert.NoError(t, json.NewEncoder(w).Encode(&marathonInfo))
```

Comment:
```suggestion
		w.Write([]byte(`{"frameworkId":"marathonId"}`))
```

Reply:
We don't need this structures since we are not testing if golang json marshal is working but if real response from marathon will be parsed properly.